extensions/topic_view_extension.rb

```diff
@@ -14,6 +14,11 @@ def self.included(base)
       unless base.const_defined?(:PRELOAD_COMMENTS_COUNT)
         base.const_set :PRELOAD_COMMENTS_COUNT, 5
       end
+
+      unless base.const_defined?(:ACTIVITY_FILTER)
```

Comment:
`if !const_defined?` ;)

Reply:
Also is this supposed to send a MessageBus message to the client or something when this changes based on this comment?

Reply:
The constant here is just holding the name of the filter when we want to sort by activity and does not require any MessageBus message to be published.